forc/src/cli/commands/init.rs

```diff
@@ -13,6 +13,18 @@ pub struct Command {
     /// Initialize a new project from a template
     #[clap(short, long, help = TEMPLATE_HELP)]
     pub template: Option<String>,
+    /// The default program type, excluding all flags or adding this flag creates a basic contract program.
+    #[clap(long)]
+    pub contract: bool,
+    /// Adding this flag creates an empty script program.
+    #[clap(long)]
+    pub script: bool,
+    /// Adding this flag creates an empty predicate program.
+    #[clap(long)]
+    pub predicate: bool,
+    /// Adding this flag creates an empty library program.
+    #[clap(long = "lib")]
```

Comment:
It feels a little odd only having a short-hand name for the `library` type but not for the others. Maybe we can omit this shorthand for now and discuss whether or not we want short-hands in another issue?